### ISSUE TITLE
Getting 404 on /webhook when installing the integration

### ISSUE THREAD

**Issue by MaximeHeckel** [danger/peril#9]
I've followed the README steps to configure peril on my laptop
Everything seems to work fine however, whenever I add the integration to a repo, ngrox logs 
```
POST /webhook                  404 Not Found
```
I guess that's what you're mentioning in
```
You need to make sure that the install message is sent to your dev setup, so it will grab the access tokens and set up an installation account for it in mongo (X-GitHub-Event: integration_installation).
```
right?
After that whenever I open a PR I get the correct payload sent to the server (with a 200 code), but nothing will show up on the pull request page despite having set up the permissions correctly (same as the Danger integration)
I guess this is due to this issue but maybe I'm wrong

**Comment by orta** [comment]
Gonna install it and set it up on a new computer right now, will update the README with my findings 💃

**Comment by MaximeHeckel** [comment]
Closing this since it has been solved in #10
